src/components/curation/GeneInformation.vue

```diff
@@ -0,0 +1,131 @@
+<script>
+export default {
+  props: {
+    geneData: {
+      type: Object,
+      required: true,
+    },
+    geneFunctionData: {
+      type: Object,
+      required: true,
+    },
+  },
+};
+</script>
+<template>
+  <div class="accordion py-1" id="gene-info-section" v-if="geneData">
+    <div class="accordion-item">
+      <h2 class="accordion-header">
+        <button
+          class="accordion-button"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#gene-info-section-body"
+          aria-expanded="true"
+          aria-controls="gene-info-section-body"
+        >
+          Gene Information
+        </button>
+      </h2>
+      <div id="gene-info-section-body" class="accordion-collapse collapse show">
+        <div class="accordion-body">
+          <div class="row">
+            <div style="width: 10%">
+              <strong><p>Symbol</p></strong>
+            </div>
+            <div style="width: 90%">
+              <router-link
+                :to="`/gene/${geneData.gene_symbol}`"
+                style="text-decoration: none"
+                v-if="geneData.gene_symbol"
+              >
+                {{ geneData.gene_symbol }}
+              </router-link>
+              <p v-else class="text-muted">Not Available</p>
+            </div>
+          </div>
+          <div class="row">
+            <div style="width: 10%">
+              <strong><p>Synonyms</p></strong>
+            </div>
+            <div style="width: 90%">
+              <p v-if="geneData.synonyms && geneData.synonyms.length > 0">
+                {{ geneData.synonyms.join(", ") }}
+              </p>
+              <p v-else class="text-muted">Not Available</p>
+            </div>
+          </div>
+          <div class="row">
+            <div style="width: 10%">
+              <strong><p>Function</p></strong>
```

Comment:
This should be renamed to "Gene product function".

Reply:
On the Gene page, it is renamed to "Function" in the latest commit, so we can keep it as "Function" here as well